File: ModuleProjects/ParkingArea/UsingJAVA/src/main/java/com/kururu/simple/project/entity/LotInformation.java
package com.kururu.simple.project.entity;

import static com.kururu.simple.project.constant.ParkingAreaEnums.DEL_FLG;

import lombok.*;

import javax.persistence.*;
import javax.validation.constraints.NotNull;
import java.io.Serializable;

/**
 * <h2>Parking Area [Entity of LOT_INFORMATION]</h2>
 * <p>Area information by Parking lot</p>
 *
 * @author Rick00Kim dreamx119@gmail.com
 */
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Entity(name = "LOT_INFORMATION")
public class LotInformation implements Serializable {

    /* Lot number(Sequence) */
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "LOT_NUMBER")
    private Long lotNumber;

    /* Lot name */
    @Column(name = "LOT_NAME",nullable = false)
    private String lotName;

    /* Acceptable small car */
    @Column(name = "ACCEPTABLE_SMALL",nullable = false)
    private Integer acceptableSmall;

    /* Acceptable medium car */
    @Column(name = "ACCEPTABLE_MEDIUM",nullable = false)
    private Integer acceptableMedium;

    /* Acceptable heavy car */
    @Column(name = "ACCEPTABLE_HEAVY",nullable = false)
    private Integer acceptableHeavy;

    /* Delete flag (Using Converter) */
    @NotNull
    @Column(name = "DEL_FLG")
    @Convert(converter = DEL_FLG.Convert.class)
    private DEL_FLG delFlg;
}

File: ModuleProjects/ParkingArea/UsingPython/parking_area/dto/lot_information.py
from sqlalchemy import Column, Integer, String
from parking_area.dto.abstract_database_dto import DatabaseDto


class LotInformation(DatabaseDto):
    """Parking Area [Entity of LOT_INFORMATION]

    """
    __tablename__ = 'LOT_INFORMATION'
    lot_number = Column('LOT_NUMBER', Integer, primary_key=True)
    lot_name = Column('LOT_NAME', String, unique=True)
    acceptable_small = Column('ACCEPTABLE_SMALL', Integer)
    acceptable_medium = Column('ACCEPTABLE_MEDIUM', Integer)
    acceptable_heavy = Column('ACCEPTABLE_HEAVY', Integer)
    del_flg = Column('DEL_FLG', String)

    def __repr__(self):
        return "<LOT_INFORMATION(lot_number='%s', lot_name='%s', acceptable_small='%d', acceptable_medium='%d',acceptable_heavy='%d', del_flg='%s')>" % (
            self.lot_number, self.lot_name, self.acceptable_small, self.acceptable_medium, self.acceptable_heavy,
            self.del_flg)
